test/e2e/lib/gutenberg/blocks/tiled-gallery-block-component.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { By } from 'selenium-webdriver';
+
+/**
+ * Internal dependencies
+ */
+import GutenbergBlockComponent from './gutenberg-block-component';
+
+class TiledGalleryBlockComponent extends GutenbergBlockComponent {
+	async uploadImages( filesDetails ) {
+		const fileInput = this.driver.findElement(
+			By.xpath( `//*[@id="${ this.blockID.slice( 1 ) }"]/div/div/div[3]/div[2]/input` )
```

Comment:
😅  Oh boy… what's this targeting? Something in the media placeholder? Maybe a dedicated helper would be good?

Reply:
This selects the hidden file input inside the tile gallery. I've moved this to its own instance field with a more descriptive name. Let me know if you have anything else in mind.